src/main/java/org/zalando/pazuzu/feature/Tag/Tag.java

```diff
@@ -0,0 +1,60 @@
+package org.zalando.pazuzu.feature.tag;
+
+import javax.persistence.*;
+
+/**
+ * Created by vpavlyshyn on 09/06/16.
+ */
+@Entity
+public class Tag {
+    public Tag(String name) {
+        this.name = name;
+    }
+    public Tag() {
+        this.name = "";
+    }
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    @Column(name = "tag_name", nullable = false, length = 256, unique = true)
+
+    private String name;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/Pazuzu/pazuzu-registry/file/2756072083/issues/source?bid=3131965&fileBranchId=3355916#l39)